crt_portal/cts_forms/templatetags/get_field_label.py

```diff
@@ -3,6 +3,17 @@
 from django.core.exceptions import FieldDoesNotExist
 register = template.Library()
 
+variable_rename = {
+    'assigned_section': 'Routed',
+    'contact_first_name': 'Contact first name',
+    'contact_last_name': 'Contact last name',
+    'location_city_town': 'City',
+    'location_state': 'State',
+    'assigned_to': 'Assignee',
+    'public_id': 'Complaint ID',
+    'primary_statute': 'Statute',
+}
+
```

Comment:
This works but may be easier to maintain if we capture these as [verbose names](https://docs.djangoproject.com/en/2.2/topics/db/models/#verbose-field-names) on the model fields themselves.